api/utils/keys/hardwarekey/cliprompt.go

```diff
@@ -93,6 +93,8 @@ func (c *cliPrompt) Touch(_ context.Context, keyInfo ContextualKeyInfo) error {
 // If an invalid PIN or PUK is provided, the user will be re-prompted until a
 // valid value is provided.
 func (c *cliPrompt) ChangePIN(ctx context.Context, _ ContextualKeyInfo) (*PINAndPUK, error) {
+	fmt.Fprintf(os.Stderr, "The default PIN %q is not supported.\n", DefaultPIN)
```

Comment:
We're already showing the same message on line 116. Maybe it’s enough to display it only when the user actually enters the default PIN?

Reply:
It sort of works as a warning against picking the default PIN, doesn't it?

Reply:
Yeah, I think the following flow would be bad UX:

```
Enter your YubiKey PIV PIN [blank to use default]:
// 123456
Please set a new 6-8 character PIN.
// 123456
The default PIN 123456 is not supported.
```